linchpin/tests/test_init_pass.py

```diff
@@ -28,6 +33,9 @@ def test_api_create():
     assert_equal(isinstance(lpa, LinchpinAPI), True)
 
 
+
```

Comment:
I think you can remove these empty lines

Reply:
Will remove them.